crates/shared/src/consts.rs

```diff
@@ -1,3 +1,4 @@
 pub const EXPECTED_RPC_VERSION: &str = "0.7.0";
 pub const RPC_URL_VERSION: &str = "v0_7";
 pub const SNFORGE_TEST_FILTER: &str = "SNFORGE_TEST_FILTER";
+pub const FREE_RPC_PROVIDER_URL: &str = "https://free-rpc.nethermind.io/sepolia-juno/v0_7";
```

Comment:
nit: Do we really want to hardcode a link to an external node in the template? This might require updates in the future. I suggest sticking with the previous devnet address `http://127.0.0.1:5050`.

Reply:
I suppose that it will be convenient for the user to have ready sepolia URL defined.

Reply:
Although it might be convenient for some users I still don't think we should include it in the template. If you add it here you are kind of responsible for it to work, which means extra effort to maintain it and potentially dealing with related support questions.